.travis.yml

```diff
@@ -87,6 +87,7 @@ matrix:
       install:
         - npm install
       script:
+        - npm list --depth=0 || cat --help
```

Comment:
Note: `npm list` throws some errors. To supress the evaluation of the return code, just executing something else.